internal/identity/identitymanager.go

```diff
@@ -105,6 +105,16 @@ func ParseKeyNormalizationConfig(strConfigVal string) int {
 // This is for cases where keys are used directly without an "author" field alongside them (custom contracts, tokens),
 // or when the author is known by the caller and should not / cannot be confirmed prior to sending (identity claims)
 func (im *identityManager) NormalizeSigningKey(ctx context.Context, inputKey string, keyNormalizationMode int) (signingKey string, err error) {
+	// In a namespace without a blockchain plugin, do not attempt to normalize.
+	if im.blockchain == nil {
+		if inputKey == "" {
+			log.L(ctx).Info("Blockchain plugin missing, falling back to using default key")
+			return im.defaultKey, nil
+		}
+		log.L(ctx).Info("Blockchain plugin missing, falling back to using input key")
+		return inputKey, nil
+	}
+
 	if inputKey == "" {
 		verifierRef, err := im.getDefaultVerifier(ctx)
```

Comment:
I solved this a bit differently, because I didn't see the benefit of adding `keyNormalizationMode` to `getDefaultVerifier`. Let me know what you think.